Article/Article.js

```diff
@@ -1,20 +1,37 @@
 // Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
+const expandText = 'Click to Expand';
+const closeText = 'Click to Close';
+
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
+    this.closeButton = this.domElement.querySelector('.closeButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = expandText;
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this));
+    this.closeButton.addEventListener('click', this.removeArticle.bind(this));
```

Comment:
This is a great use of bind.
 If you like you can as well use arrow functions to "skip" this passage and do it automatically in this fashion : `  this.expandButton.addEventListener("click", () => this.expandArticle());`